### PR TITLE
Ecosystem website: design updated

### LINKED ISSUE #574: Design feedback on new static website
## Change the logo

> removing the Qiskit logo from the front of the word mark, and changing some colors. So it is just “Qiskit Ecosystem” 

## Color scheme

> then the buttons and links should be [Carbon] blue 60. 

> I would also suggest some color updates to the tagging scheme, but I am happy to mock this up over the next week

### PR DESCRIPTION
**Pull request by arnaucasau**
### Summary

This PR updates the design of the ecosystem webpage, following the changes detailed in #574. Thank you, @mackinziblank, for the cool changes and the mockups!

Link to preview the changes: https://arnaucasau.github.io/ecosystem/

Closes #574

### CODE REVIEW

File: website/style.css [hunk 1]
```diff
@@ -15,16 +15,31 @@ body {
...
+
+.ecosystem-title {
+  font-size: 60px;
+  line-height: 70px;
+  font-weight: 300;
 }
 
-.ecosystem-logo {
-  width: 350px;
+.qiskit {
```
Comment (Eric-Arellano) [inline website/style.css:41]: this is a bit vague. Maybe something like `header-qiskit-text` or `header-qiskit-word`.

File: website/style.css [hunk 2]
```diff
@@ -15,16 +15,31 @@ body {
...
+.header-content{
+  width: fit-content;
+}
+
+.header-logo {
+  margin-right: 48px;
+}
+
+.ecosystem-title {
+  font-size: 60px;
```
Comment (Eric-Arellano) [inline website/style.css:36]: I think this should probably use `rem`: https://carbondesignsystem.com/guidelines/typography/overview/#scale

**Review by Eric-Arellano — APPROVED** [review]
This looks amazing! I'm approving the code but please wait till you get sign off from a designer.

**Review by Eric-Arellano — APPROVED** [review]
Feel free to merge once the designers approve :)

**Review by frankharkins — APPROVED** [review]
Thanks @arnaucasau! Page looks really polished 🚀

**Pull request merged**